packages/example-app-base/src/components/Page/Page.module.scss

```diff
@@ -2,6 +2,7 @@
 @import '../../styles/mixins';
 
 $PageSubHeadMargin: 20px;
+$usageListPadding: 24px;
```

Comment:
Combined the other files into this one so we don't get the contents of this file duplicated x6